src/main/java/karuna/karuna_backend/security/config/GlobalAuthExceptionHandler.java

```diff
@@ -1,12 +1,12 @@
-package karuna.karuna_backend.Config;
+package karuna.karuna_backend.security.config;
```

Comment:
Same here, it should be moved either way to User -> infrastrcture since that handles User Service exceptions I believe. We could merge ExceptionHandler with the one that handles UserNowFound Exception (from exception package).